cmd/influxd/run/server_suite_test.go

```diff
@@ -19,13 +19,13 @@ func init() {
 			&Query{
 				name:    "create database should succeed",
 				command: `CREATE DATABASE db0`,
-				exp:     `{"results":[{}]}`,
+				exp:     `{"results":[{"statement_id":0}]}`,
```

Comment:
Since the `statement_id` is a generated ID, and doesn't matter if there are no results, the field should probably have `omitempty` added to the struct tag as well. That would preserve the existing "empty object" result, and still allow for tracking partial series results